ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java

```diff
@@ -725,19 +725,45 @@ public void onFocusChange(View v, boolean hasFocus) {
         new TextView.OnEditorActionListener() {
           @Override
           public boolean onEditorAction(TextView v, int actionId, KeyEvent keyEvent) {
-            // Any 'Enter' action will do
-            if ((actionId & EditorInfo.IME_MASK_ACTION) > 0 ||
+            // Any 'Enter' action will do, except next and previous
```

Comment:
Instead of comment I would do variable like "Clean code" book suggest:
```
boolean isAnyEnterActionOtherThanNextOrPrevious = (
  (actionId & EditorInfo.IME_MASK_ACTION) > 0 &&
  actionId != EditorInfo.IME_ACTION_NEXT &&
  actionId != EditorInfo.IME_ACTION_PREVIOUS
) || actionId == EditorInfo.IME_NULL;
if (isAnyEnterActionOtherThanNextOrPrevious) {
 // code
}


```